source/getting-started/components/wazuh-dashboard.rst

```diff
@@ -56,37 +61,42 @@ The Wazuh dashboard provides a user interface dedicated to manage your Wazuh dep
 .. hlist::
    :columns: 2
 
-   -  .. thumbnail::  /images/getting-started/dashboard_administration.png 
+   -  .. thumbnail::  /images/getting-started/platform-management.png 
          :align: center
          :width: 100% 
-         :title: Administration
+         :title: Platform management
+         :alt: Platform management
 
-   -  .. thumbnail::  /images/getting-started/dashboard_status.png 
+   -  .. thumbnail::  /images/getting-started/status-and-reports.png 
          :align: center
          :width: 100% 
          :title: Status and reports
+         :alt: Status and reports
 
 Developer tools
 ---------------
 
 The Wazuh dashboard includes a Ruleset Test tool that can process log messages to check how it is decoded and if it matches a threat detection rule or not. This feature is especially useful when custom decoders and rules have been created and the user wants to test them.
 
-.. thumbnail::  /images/getting-started/dashboard_ruleset_test.png 
+.. thumbnail::  /images/getting-started/developer-tools.png 
```

Comment:
This image is showing specifically the ruleset testing tool, which is, like the API console, one of the available developer tools